web/pandas/pdeps/0014-string-dtype.md

```diff
@@ -215,16 +215,16 @@ and NaN semantics.
 Overview of the different ways to specify a dtype and the resulting concrete
 dtype of the data:
 
-| User specification                     | Concrete dtype                                    | String alias            | Note |
-|----------------------------------------|---------------------------------------------------|-------------------------|------|
-| Unspecified (inference)                | `StringDtype(storage="pyarrow"|"python", na_value=np.nan)` | "string"                | (1)  |
-| `StringDtype()` or `"string"`              | `StringDtype(storage="pyarrow"|"python", na_value=np.nan)` | "string"                | (1), (2)  |
-| `StringDtype("pyarrow")`                 | `StringDtype(storage="pyarrow", na_value=np.nan)`           | "string"                | (2)     |
-| `StringDtype("python")`                  | `StringDtype(storage="python", na_value=np.nan)`            | "string"                | (2)     |
-| `StringDtype("pyarrow", na_value=pd.NA)` | `StringDtype(storage="pyarrow", na_value=pd.NA)`            | "string[pyarrow]"       |      |
-| `StringDtype("python", na_value=pd.NA)`  | `StringDtype(storage="pyarrow", na_value=pd.NA)`            | "string[python]"        |      |
-| `StringDtype(na_value=pd.NA)`            | `StringDtype(storage="pyarrow"|"python", na_value=pd.NA)` | "string[pyarrow]" or "string[python]" | (1)  |
-| `StringDtype("pyarrow_numpy")`           | `StringDtype(storage="pyarrow", na_value=np.nan)`           | "string[pyarrow_numpy]" | (3)  |
+| User specification                       | Concrete dtype                                                | String alias                          | Note     |
+|------------------------------------------|---------------------------------------------------------------|---------------------------------------|----------|
+| Unspecified (inference)                  | `StringDtype(storage="pyarrow"\|"python", na_value=np.nan)`   | "string"                              | (1)      |
+| `StringDtype()` or `"string"`            | `StringDtype(storage="pyarrow" \| "python", na_value=np.nan)` | "string"                              | (1), (2) |
+| `StringDtype("pyarrow")`                 | `StringDtype(storage="pyarrow", na_value=np.nan)`             | "string"                              | (2)      |
+| `StringDtype("python")`                  | `StringDtype(storage="python", na_value=np.nan)`              | "string"                              | (2)      |
+| `StringDtype("pyarrow", na_value=pd.NA)` | `StringDtype(storage="pyarrow", na_value=pd.NA)`              | "string[pyarrow]"                     |          |
+| `StringDtype("python", na_value=pd.NA)`  | `StringDtype(storage="python", na_value=pd.NA)`               | "string[python]"                      |          |
+| `StringDtype(na_value=pd.NA)`            | `StringDtype(storage="pyarrow" \| "python", na_value=pd.NA)`  | "string[pyarrow]" or "string[python]" | (1)      |
```

Comment:
Shouldn't this include `"String"` as the string alias?  

If the goal is to just list the string alias that we show as the `dtype` of the column,  it's fine, but maybe the "String alias" column should say "String representation of dtype" and then you have a column called "User string alias" that would include all the aliases a user could use to specify the `dtype`, which would include `"String"`

In addition, maybe consider that the NA variants return `"String[pyarrow]"` or `"String[python]"` as the `dtype` which makes it clear that it is using the `pd.NA` variant via capitalization.

Reply:
Yeah, I wasn't entirely sure how to integrate `"String"` without making the table even more complex ..

Now, I can indeed change  `"string[python]"` / `"string[pyarow]"` to the capitalized version `"String[python]"` / `"String[python]"`, and mention `"String"` in the first column as alias for `pd.StringDtype(na_value=pd.NA)`

I was originally planning to keep `"string[python]"` / `"string[pyarow]"` mean what it does today for backwards compatibility (and since I mention that this is not allowed for the new NaN-variant of the dtype). Now that I updated the proposal to use capitalized `"String"` for the NA-variant, that of course makes things a bit confusing if the lower case versions with explicit storage in the alias still means the NA-variant ..